apps/web/src/App.tsx

```diff
@@ -50,7 +52,7 @@ export default function App() {
           <RecordingNotifications />
           <MobileTopMenuBar />
           <Room />
-          <MenuBar />
+          {isMobile ? <MobileBottomMenuBar /> : <MenuBar />}
```

Comment:
I think we should try to just use `<MenuBar />` only, but if we had `<MobileBottomMenuBar />` too then we should use this to switch between the two:
```
  const isMobile = useMediaQuery((theme: Theme) => theme.breakpoints.down('sm'));
```
`isMobile` from `utils` just looks at the user agent. For styling on mobile, we should use media queries.